### PR TITLE
The Defib Nerf (Defibs are no longer a 10 second paralyze)

### PR DESCRIPTION
**Pull request by carlarctg**
## About The Pull Request
Defib stun now knocks down for 7.5 seconds.
Adds a new status effect, 'convulsing'.
Convulsion makes you drop your inhand item with a 40% chance every time Life is triggered.
Defib gives you 15 seconds of convulsion.

## Why It's Good For The Game
Defibs survived the stun baton purge of hardstunning. They instantly stun you for 10 seconds and have a 5 second recharge time, not to mention they deal 50 stamina damage and can for some reason stun borgs. 
I wanted to keep them as being a superior choice over the stun baton, as they take up a belt slot or back slot, two hands, and preparation. Simply making them a shitty stun baton would mean that everyone would just make a stunprod or stunbaton without bothering to emp/emag the defib whatsoever, ruining the point of their existence.

## Changelog
:cl:
balance: Emagged defillibrators now knock people down for 7.5 seconds, deal 55 stamina damage, and convulse for 15 seconds, They don't stun borgs anymore.
add: New status effect: Convulsions. 40% chance to drop your inhand item every tick.
balance: Harm intent defillibrator heart attack takes half as much time.
fix: ERT preproom no longer has a syndicate defillibrator.
add: It now has a nanotrasen elite defillibrator, extremely high quality and definitely not a repaint of a syndicate defib.
/:cl:

### CODE REVIEW

**Comment by CRITAWAKETS** [comment]
Massive antag nerf for stealth ops, as combat defibbing someone to stun them before using an execution weapon of choice (Most often CQC) is something that is done often by experienced players.

**Comment by carlarctg** [comment]
@CRITAWAKETS Combat defibs have a doubled recharge rate, and you're trying to defend stealth ops, so whatever.

**Comment by 81Denton** [comment]
>Massive antag nerf for stealth ops

From what I've seen it's mostly used by people who EMP a regular defib and zap people on disarm attempt.

**Comment by Tharcoonvagh** [comment]
Ignore the combat defib, we dont need another stealth op nerf

**Comment by Tlaltecuhtli** [comment]
either make a stunbaton like or make it use electrocute

**Comment by HomocidlePancakes** [comment]
Does it still stun borgs? I hope it does.

**Comment by 4dplanner** [comment]
> ~~Ignore the combat defib, ~~we dont need ~~another ~~stealth ops~~ nerf~~

**Comment by Iatots** [comment]
>person with blood red combat equipment exists
>shoot them first

WHAT THE FUCK NOOOO HOW DID YOU KNOW I AM VALID I COULD HAVE FOUND THIS IN THE CLOWNS POCKET NOOO MY ANTAG ROLL!!! 
>don't shoot first

HAHA GET DUNKED ON YOU IDIOT!!! I AM SO FUCKING ROBUST OH MY GOD!! STEALTH OPS GANG!!!

**Comment by necromanceranne** [comment]
You people know CQC exists right?

**Comment by wesoda25** [comment]
People getting mad bc blitz ops won't be braindead easy anymore lol

**Comment by carlarctg** [comment]
> People getting mad bc blitz ops won't be braindead easy anymore lol

implying anyone ever used the combat defib to stun

**Comment by PKPenguin321** [comment]
can somebody explain how in gods name youre supposed to give somebody a heart attack with a hacked defib if it takes 10 seconds of no motion to do and there isn't a single fucking stun in the game taht doesnt let the victim beat you to death from the ground?

**Comment by optimumtact** [comment]
tactical defibs are not a thing, go away bf badcompany medic players

**Comment by CRITAWAKETS** [comment]
Huh. Everyone hates stealth and blitz ops. Might aswell make War Ops the only mode if ultimately everyone's going to always do it and prefers doing it.

**Comment by carlarctg** [comment]
> can somebody explain how in gods name youre supposed to give somebody a heart attack with a hacked defib if it takes 10 seconds of no motion to do and there isn't a single fucking stun in the game taht doesnt let the victim beat you to death from the ground?

stamina damage stuns for 10 seconds

this used to stun for 10s too so

if you want, i can reduce the heart attack time to 8 or 9 so it's actually possible

**Comment by necromanceranne** [comment]
The heart attack induction also is certain death that cannot be easily remedied  by typical med doping and powerful regeneration abilities like healing viruses and nanies. There is a purpose to it on using it to kill crit players if you know they have these things. In war ops, that's pretty likely now that nanites research separately to normal research.

**Comment by wesoda25** [comment]
> Huh. Everyone hates stealth and blitz ops. Might aswell make War Ops the only mode if ultimately everyone's going to always do it and prefers doing it.

Better yet we could remove war ops

**Comment by carlarctg** [comment]
> > Huh. Everyone hates stealth and blitz ops. Might aswell make War Ops the only mode if ultimately everyone's going to always do it and prefers doing it.
> 
> Better yet we could remove war ops

ah yes. you're right. let's remove the gamemode where everyone knows what's going to happen, and can use 20 minutes to prepare accordingly, making medicine, ordering guns, mining for ores, making bombs for research, planting medicinal plants, etcetera

instead, let's keep the one where the captain dies in 10 minutes to a stimpack AP l6 saw nukie every **single** round, because even if they're wary for the first 10 minutes and hang around near secured areas the nukies can simply emag in and use the instakill tools they have to secure the disk inmediately

or, better yet, stealth ops! where it's a greenshift for 10 to 30 minutes until one of the nukies finally works up the courage to run up to the captain and shoot him while he's standing AFK in the hallways, ending the round prematurely and making everyone's efforts to do anything in the round meaningless and wasted (which also happen in blitz ops)

**Comment by optimumtact** [comment]
you're right we should remove nuclear ops

**Comment by PKPenguin321** [comment]
>if you want, i can reduce the heart attack time to 8 or 9 so it's actually possible

do it

**Comment by wesoda25** [comment]
war ops is such insufferable garbage id rather the station get nuked without warning after 10 minutes than suffer through the carbon copy each war op round is

File: code/modules/mob/status_procs.dm [hunk 1]
```diff
@@ -4,6 +4,9 @@
 // eye damage, eye_blind, eye_blurry, druggy, TRAIT_BLIND trait, and TRAIT_NEARSIGHT trait.
 
 
+//hand convulsionies
+/mob/proc/convulsing(amount)
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/mob/status_procs.dm]: use status effects what the frick?

Comment (ExcessiveUseOfCobblestone) [reply code/modules/mob/status_procs.dm]: as in https://github.com/tgstation/tgstation/blob/0b03db27bff11b2276ed0eb879360baa8bb96512/code/datums/status_effects/status_effect.dm

**Comment by ExcessiveUseOfCobblestone** [comment]
> this kills stealth ops

stealth ops don't use a defib because the one available to them is a different icon...

File: code/modules/mob/status_procs.dm (hunk 1 above)
Comment (carlarctg) [reply code/modules/mob/status_procs.dm]: reeee why

File: code/game/objects/items/defib.dm [hunk 2]
```diff
@@ -483,12 +483,14 @@
 	busy = TRUE
 	M.visible_message("<span class='danger'>[user] has touched [M] with [src]!</span>", \
 			"<span class='userdanger'>[user] has touched [M] with [src]!</span>")
-	M.adjustStaminaLoss(50)
-	M.Paralyze(100)
-	M.updatehealth() //forces health update before next life tick //isn't this done by adjustStaminaLoss anyway?
+	M.adjustStaminaLoss(60)
+	M.Knockdown(75)
+	M.Jitter(50)
+	M.convulsing(150) //makes your hands spaz out and drop items every two lifes or so
```
Comment (ATH1909) [inline code/game/objects/items/defib.dm]: Note that a tick of Life() takes place every TWO seconds, not every second, so convulsing isn't as strong of a status effect as you might think that it is.

File: code/modules/mob/living/carbon/life.dm [hunk 3]
```diff
@@ -483,6 +483,15 @@ GLOBAL_LIST_INIT(ballmer_windows_me_msg, list("Yo man, what if, we like, uh, put
 	else
 		SEND_SIGNAL(src, COMSIG_CLEAR_MOOD_EVENT, "jittery")
 
+	if(convulsion)
+		if(prob(40))
+			var/obj/item/I = get_active_held_item()
+			if(I && dropItemToGround(I))
+				to_chat(src, "<span class='userdanger'>Your hand convulses violently and you drop what you were holding!</span>")
```
Comment (ATH1909) [inline code/modules/mob/living/carbon/life.dm]: maybe add a comment after "violently"?

File: code/game/objects/items/defib.dm (hunk 2 above)
Comment (carlarctg) [reply code/game/objects/items/defib.dm]: you're underestimating being almost unable to use items

**Comment by carlarctg** [comment]
@ExcessiveUseOfCobblestone made convulsions a status effect

also went ahead and fixed the issue where ERTs had a syndicate defib in their preparation room...by adding a whole new defib

File: code/modules/mob/living/carbon/life.dm (hunk 3 above)
Comment (carlarctg) [reply code/modules/mob/living/carbon/life.dm]: 👍

File: code/game/objects/items/defib.dm [hunk 4]
```diff
@@ -508,7 +517,7 @@
 		"<span class='warning'>You overcharge the paddles and begin to place them onto [H]'s chest...</span>")
 	busy = TRUE
 	update_icon()
-	if(do_after(user, 30, target = H))
+	if(do_after(user, 15, target = H))
```
Comment (ExcessiveUseOfCobblestone) [inline code/game/objects/items/defib.dm]: undoc'd?

File: code/game/objects/items/defib.dm [hunk 5]
```diff
@@ -517,7 +526,7 @@
 			T.audible_message("<span class='warning'>\The [defib] lets out an urgent beep and lets out a steadily rising hum...</span>")
 		else
 			user.audible_message("<span class='warning'>[src] let out an urgent beep.</span>")
-		if(do_after(user, 30, target = H)) //Takes longer due to overcharging
+		if(do_after(user, 15, target = H)) //Takes longer due to overcharging
```
Comment (ExcessiveUseOfCobblestone) [inline code/game/objects/items/defib.dm]: doc?

**Review by ExcessiveUseOfCobblestone — CHANGES_REQUESTED** [review]

File: code/game/objects/items/defib.dm (hunk 5 above)
Comment (carlarctg) [reply code/game/objects/items/defib.dm]: oh yeah i implemented what PK wanted but forgot to document it. whoops

Comment (carlarctg) [reply code/game/objects/items/defib.dm]: > doc?

docced

**Review by kriskog — CHANGES_REQUESTED** [review]
Delete the now unused defibcombat-emagged icon state, and this is good to go

File: code/game/objects/items/defib.dm [hunk 6]
```diff
@@ -678,13 +687,20 @@
 
 /obj/item/twohanded/shockpaddles/syndicate
 	name = "syndicate defibrillator paddles"
-	desc = "A pair of paddles used to revive deceased operatives. It possesses both the ability to penetrate armor and to deliver powerful or disabling shocks offensively."
+	desc = "A pair of paddles used to revive deceased operatives. They possesses both the ability to penetrate armor and to deliver powerful or disabling shocks offensively."
```
Comment (ATH1909) [inline code/game/objects/items/defib.dm]: "they possesses"

Comment (kriskog) [inline code/game/objects/items/defib.dm]:
```suggestion
	desc = "A pair of paddles used to revive deceased operatives. They possess both the ability to penetrate armor and to deliver powerful or disabling shocks offensively."
```

File: code/game/objects/items/defib.dm [hunk 7]
```diff
@@ -678,13 +687,20 @@
...
 	combat = TRUE
 	icon = 'icons/obj/defib.dmi'
 	icon_state = "syndiepaddles0"
 	item_state = "syndiepaddles0"
 	base_icon_state = "syndiepaddles"
 
+/obj/item/twohanded/shockpaddles/syndicate/nanotrasen
+	name = "elite nanotrasen defibrillator paddles"
+	desc = "A pair of paddles used to revive deceased ERT members. They possesses both the ability to penetrate armor and to deliver powerful or disabling shocks offensively."
```
Comment (kriskog) [inline code/game/objects/items/defib.dm]:
```suggestion
	desc = "A pair of paddles used to revive deceased ERT members. They possess both the ability to penetrate armor and to deliver powerful or disabling shocks offensively."
```

**Pull request merged**